packages/istanbul-reports/test/fixtures/github-649.json

```diff
@@ -100,11 +100,6 @@
             "line": 6,
             "column": 5
           }
-        }, {
```

Comment:
I'm not understanding why you deleted this from your fixture?

Should we also handle the empty `start`/`end` use case?